<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отсечение отрезков выпуклым отсекателем</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>602</width>
      <height>602</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отобразить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="funcs">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="x_angle">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="y_angle">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>470</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="z_angle">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="step_x">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="step_z">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_x">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>-5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_x">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_z">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>-5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_z">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>290</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
